tuya.cpp

```diff
@@ -955,3 +1057,60 @@ bool DeRestPluginPrivate::SendTuyaRequest(TaskItem &taskRef, TaskType taskType ,
 
     return true;
 }
+
+bool DeRestPluginPrivate::SendTuyaCommand( const deCONZ::ApsDataIndication &ind, qint8 command, QByteArray data )
```

Comment:
Don't add spaces before and after braces. &rarr; Consistency with the the rest of the code base.
Same goes for all other statements like if, for, ...